app/src/main/java/com/lightricks/feedexercise/data/database/Entities.kt

```diff
@@ -0,0 +1,22 @@
+package com.lightricks.feedexercise.data.database
+
+import androidx.room.Entity
+import androidx.room.Index
+import androidx.room.PrimaryKey
+import com.lightricks.feedexercise.data.database.FeedEntity.Companion.TABLE_NAME
+
+@Entity(
+    tableName = TABLE_NAME,
+    indices = [Index("feedItemId")]
+)
+data class FeedEntity(
+    @PrimaryKey(autoGenerate = true)
+    val id: Int = 0,
```

Comment:
Why do we need it if we have already the second id (feedItemId)?

Reply:
I prefer not to rely feedItemId which we get from the server.

Reply:
I would index the feedItemId field.